.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+Closes # .
+
+<!-- Reference the issue corresponding to this PR. If an issues does not exist, consider opening one or writing a detailed description for what this PR changes and it's value in the following sections. -->
```

Comment:
I'm favor of a stronger language (as explained above inside the contributing guidelines not as PR template). Something along the lines of "Unless it is a trivial bug fix or documentation improvements, please open an issue first discussing the change with us. Not doing so, risks us not accepting the PR and wasting your time."